anesthetic/plot.py

```diff
@@ -402,6 +392,79 @@ def contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
     return cbar
 
 
+def hist_plot_2d(ax, data_x, data_y, *args, **kwargs):
+    """Plot a 2d marginalised distribution as a histogram.
+
+    This functions as a wrapper around matplotlib.axes.Axes.hist2d
+
+    Parameters
+    ----------
+    ax: matplotlib.axes.Axes
+        axis object to plot on
+
+    data_x, data_y: numpy.array
+        x and y coordinates of uniformly weighted samples to generate kernel
+        density estimator.
+
+    xmin, xmax, ymin, ymax: float
+        lower/upper prior bounds in x/y coordinates
+        optional, default None
+
+    contours: list
+        amount of mass within each iso-probability contour.
+        If None defaults to hist2d.
+        Default [0.05, 0.33]
```

Comment:
It's not really the amount of mass within the contour, right? I found this confusing in `contour_plot_2d` as well, as I am more used to the numbers 0.68 and 0.95. This is actually more what `plt.contour` refers to as `levels`. The amount of mass within the contours is really one minus the levels.

Reply:
OK, I think I've improved on this now, so that the numbers correspond to the intuitive values of 0.68 and 0.95